bilge-impl/src/bitsize.rs

```diff
@@ -223,40 +181,6 @@ fn modify_special_field_names(fields: &mut Fields) {
     }
 }
 
-fn analyze_struct(fields: &Fields) -> TokenStream {
-    if fields.is_empty() {
-        abort_call_site!("structs without fields are not supported")
-    }
-
-    // NEVER move this, since we validate all nested field types here as well.
```

Comment:
moving all the other checks to `internal` was probably a good idea, but we still have to validate the field types like I noted here.
that rhymes

Reply:
hope it's okay now.